src/components/profile/ProfileService.js

```diff
@@ -262,11 +298,47 @@ goog.require('ga_urlutils_service');
               .attr('class', 'ga-profile-label ga-profile-label-y')
               .attr('transform', 'rotate(-90)')
               .attr('y', 0 - options.margin.left)
-              .attr('x', 0 - height / 2 - 20)
+              .attr('x', 0 - height / 2 - 30)
               .attr('dy', '1em')
+              .attr('font-size', '0.95em')
               .text($translate.instant(options.yLabel) + ' [m]');
 
-           return element;
+          //Total Elevation Difference
+          group.append('text')
+              .attr('class', 'ga-profile-label ga-profile-elevation-difference')
+              .attr('x', 0)
+              .attr('y', height + options.margin.bottom - 1)
```

Comment:
I know that it's a detail, but it gives one pixel space below the labels:
![-1](https://cloud.githubusercontent.com/assets/14076451/10516628/76fab9f8-7359-11e5-95d7-e84ba1d73224.PNG)